api/api/clubs/views.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Club
+from .serializers import ClubsSerializer
+
+
+class ClubsDetails(APIView):
+    def get(self, request):
+        queryset = Club.objects.all()
+        serializer = ClubsSerializer(queryset, many=True)
+        return Response(serializer.data)
```

Comment:
I believe, we won't need this method if we extend the `ListApiView`

This should do:
```python
class ListClubsView(ListApiView):
    queryset = Clubs.objects.all()
    serializer_class = ClubsSerializer
```